packages/fastify-renderer/src/node/renderers/Renderer.ts

```diff
@@ -42,10 +42,10 @@ export interface Renderer {
   vitePlugins(): Plugin[]
 }
 
-export function scriptTag(render: Render, content: string, attrs: Record<string, string> = {}) {
-  if ('cspNonce' in render.reply) {
-    attrs.nonce ??= (render.reply as any).cspNonce.script
-  }
+export function scriptTag(content: string, attrs: Record<string, string> = {}) {
+  // if ('cspNonce' in render.reply) {
+  //   attrs.nonce ??= (render.reply as any).cspNonce.script
+  // }
```

Comment:
These checks don't work anymore because fastify isn't available in the context in which we run hooks anymore.

Reply:
Actually, I have some ideas on how to make it work. We can pass the individual attributes directly without references to `render`.

Reply:
Ok -- i think these CSP nonces are quite important so we definitely need a way to support that.

Reply:
Done!